pkgs/development/libraries/onnxruntime/default.nix

```
@@ -19,6 +19,8 @@
 , gtest
 , protobuf3_21
 , pythonSupport ? true
+, cudaSupport ? true
```

```suggestion
, cudaSupport
```

and then use this pattern: https://github.com/NixOS/nixpkgs/blob/d48c7318283b64de6a41b2c68e28fc292a755ca0/pkgs/top-level/python-packages.nix#L5532